clickhouse_connect/common.py

```diff
@@ -59,3 +60,4 @@ def _init_common(name: str, options: Sequence[Any], default: Any):
 _init_common('dict_parameter_format', ('json', 'map'), 'json')
 _init_common('invalid_setting_action', ('send', 'drop', 'error'), 'error')
 _init_common('product_name', (), '')
+_init_common('readonly', (0, 1, 2), 0)
```

Comment:
I think we could limit this to 0 and 1 since it's only necessary for older versions

Reply:
done